MediaGalleryUi/Model/Config.php

```diff
@@ -9,11 +9,12 @@
 namespace Magento\MediaGalleryUi\Model;
 
 use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\MediaGalleryUiApi\Model\ConfigInterface;
```

Comment:
@sivaschenko .. I think we should move `ConfigInterface` from `Model` folder to `Api` folder

Reply:
Yes